sdk/search/search-documents/tests.yml

```diff
@@ -5,4 +5,6 @@ stages:
     parameters:
       PackageName: "@azure/search-documents"
       ServiceDirectory: search
-      SupportedClouds: 'Public,UsGov,China'
+      # TODO: change 'Preview' cloud back to public after search RP fixes deletion metadata issue
```

Comment:
Lets please add a link to an issue tracking this todo.

Reply:
Ack, had auto-merge on. I'm tracking at https://github.com/Azure/azure-sdk-tools/issues/2216 though.